### PR TITLE
Notification to open folder after save

### LINKED ISSUE #204: Option to open folder after save
I will love to see a setting for open the destination folder after save.

### PR DESCRIPTION
**Pull request by karaggeorge**
Fixes #204

Didn't know what the text (title, body) should be. Gave it a try

## Gif
kapture 2017-12-11 at 18 12 59

### CODE REVIEW

**Comment by albinekb** [comment]
Nice! But not quite correct to the issue description. There should be an option in the share-service extension to enable opening the file automatically without any notification I think? 🙂

File: app/src/main/save-file-service.js [hunk 1]
```diff
@@ -32,6 +32,17 @@ const action = async context => {
 
   // TODO: Switch to the async version when we target Electron 1.8
   moveFile.sync(await context.filePath(), filePath);
+
+  const notification = new Notification({
+    title: 'File Saved Successfully',
+    body: 'Click to open'
```
Comment (sindresorhus) [inline app/src/main/save-file-service.js]: You're not opening the file, you're revealing it.

> Click to show the file in Finder

File: app/src/main/save-file-service.js [hunk 2]
```diff
@@ -32,6 +32,17 @@ const action = async context => {
 
   // TODO: Switch to the async version when we target Electron 1.8
   moveFile.sync(await context.filePath(), filePath);
+
+  const notification = new Notification({
+    title: 'File Saved Successfully',
```
Comment (sindresorhus) [inline app/src/main/save-file-service.js]: > File saved successfully!

**Comment by karaggeorge** [comment]
@sindresorhus wording updated 👍

**Comment by sindresorhus** [comment]
Great! Thanks for all the high-quality PRs, @karaggeorge :) Keep 'em coming.

**Pull request merged**